test/plots/segmentPlotTests.ts

```diff
@@ -2,159 +2,168 @@
+      it("autorangeMode(\"x\")", () => {
+        let staggeredData = [
+          { y: 0, x: 0, x2: 1 },
+          { y: 1, x: 1, x2: 2 }
+        ];
+        xScale.padProportion(0);
 
-      yScale.domain([0.5, 1.5]);
-      assert.deepEqual(xScale.domain(), [1, 2], "y domain includes only the visible segment (second)");
+        let plot = new Plottable.Plots.Segment();
+        plot.x((d) => d.x, xScale);
+        plot.x2((d) => d.x2);
+        plot.y((d) => d.y, yScale);
+        plot.addDataset(new Plottable.Dataset(staggeredData));
+        plot.autorangeMode("x");
 
-      svg.remove();
-    });
+        plot.renderTo(svg);
 
-    it("autorangeMode(\"y\")", () => {
-      let staggeredData = [
-        { x: 0, y: 0, y2: 1 },
-        { x: 1, y: 1, y2: 2 }
-      ];
-      yScale.padProportion(0);
+        assert.deepEqual(xScale.domain(), [0, 2], "y domain includes both visible segments");
+
+        yScale.domain([-0.5, 0.5]);
+        assert.deepEqual(xScale.domain(), [0, 1], "y domain includes only the visible segment (first)");
 
-      let plot = new Plottable.Plots.Segment();
-      plot.x(function(d) { return d.x; }, xScale);
-      plot.y(function(d) { return d.y; }, yScale);
-      plot.y2(function(d) { return d.y2; });
-      plot.addDataset(new Plottable.Dataset(staggeredData));
-      plot.autorangeMode("y");
-      plot.renderTo(svg);
+        yScale.domain([0.5, 1.5]);
+        assert.deepEqual(xScale.domain(), [1, 2], "y domain includes only the visible segment (second)");
+      });
+
+      it("autorangeMode(\"y\")", () => {
+        let staggeredData = [
+          { x: 0, y: 0, y2: 1 },
+          { x: 1, y: 1, y2: 2 }
+        ];
+        yScale.padProportion(0);
 
-      assert.deepEqual(yScale.domain(), [0, 2], "y domain includes both visible segments");
+        let plot = new Plottable.Plots.Segment();
+        plot.x((d) => d.x, xScale);
+        plot.y((d) => d.y, yScale);
+        plot.y2((d) => d.y2);
+        plot.addDataset(new Plottable.Dataset(staggeredData));
+        plot.autorangeMode("y");
 
-      xScale.domain([-0.5, 0.5]);
-      assert.deepEqual(yScale.domain(), [0, 1], "y domain includes only the visible segment (first)");
+        plot.renderTo(svg);
 
-      xScale.domain([0.5, 1.5]);
-      assert.deepEqual(yScale.domain(), [1, 2], "y domain includes only the visible segment (second)");
+        assert.deepEqual(yScale.domain(), [0, 2], "y domain includes both visible segments");
 
-      svg.remove();
+        xScale.domain([-0.5, 0.5]);
+        assert.deepEqual(yScale.domain(), [0, 1], "y domain includes only the visible segment (first)");
+
+        xScale.domain([0.5, 1.5]);
+        assert.deepEqual(yScale.domain(), [1, 2], "y domain includes only the visible segment (second)");
+      });
     });
 
-    describe("entitiesIn() returns segments that intersect with the given constraints", () => {
+    describe("SegmentPlot - entitiesIn()", () => {
```

Comment:
It's nested inside the a `describe("SegmentPlot)"`, so it's probably not worth repeating the `Plot` type here.